core/templates/dev/head/domain/classifier/ClassifierObjectFactorySpec.js

```diff
@@ -27,7 +27,7 @@ describe('Classifier Object Factory', function() {
 
   it('should create a new classifier', function() {
     var classifierObject = (
-     ClassifierObjectFactory.create(
+      ClassifierObjectFactory.create(
         'TestClassifier', {}, 1));
```

Comment:
Will this line be under the line length limit if you move the args to the line above? If so, that would be preferred.